docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp
+    :language: c++
+    :lines: 82-85
+    :dedent: 4
+
+
+In the constructor, the pointers are populated by calling the APIs provided by the rmw and rmw_fastrtps_cpp, until obtaining the `eprosima::fastdds::dds::DataWriter` handle:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp
+    :language: c++
+    :lines: 42-45
+    :dedent: 4
+
+
+When the `pubX_ownership_strength` is updated (for instance, via command line using `ros2 param set` command), the `cb` parameter callback is raised, and the `eprosima::fastdds::dds::DataWriter` handle is used to update its ownership strength.
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/src/change_mutable_qos_pub1.cpp
+    :language: c++
+    :lines: 61-70
+    :dedent: 8
+
+In this case, as in the current version of Fast DDS the builtin statistics are enabled by default (see `DomainParticipantQos <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domainParticipant/domainParticipant.html#domainparticipantqos>`_), it is needed to retrieve the internal QoS by means of `::get_qos()`, then perform the modifications and update the QoS by means of `::set_qos()`:
+The value of the ownership strength is set from the value of the updated parameter.
+
+Update CMakeLists.txt and package.xml
+-------------------------------------
+
+We need to add the instructions to compile the new source files, and to account for its dependencies both in CMakeLists.txt and package.xml files.
+
+Make sure that the find_package lines in the CMakeLists.txt are the following, so substitute what you have for the following lines:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/CMakeLists.txt
+    :language: ccmake
+    :lines: 9-12
+
+
+Then add the following lines to compile and install each node:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/CMakeLists.txt
+    :language: ccmake
+    :lines: 14-36
+
+
+Inside package.xml file, make sure that the <depend> tags, are the following, so substitute what you have for the following lines:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/vulcanexus_change_mutable_qos/package.xml
+    :language: xml
+    :lines: 12-14
+
+
+Configure initial QoS
+---------------------
+
+Ownership Strength Policy is mutable, but Ownership Policy is not. Then, it is needed to configure EXCLUSIVE_OWNERSHIP_POLICY to all participants before running the ROS nodes.
+To do that, create a new xml file in the root of the workspace:
+
+.. code-block:: bash
+
+    cd ~/vulcanexus_ws
+    touch profiles1.xml
+
+Open the newly created file with your preferred editor and paste the following xml code:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/profiles1.xml
```

Comment:
These XMLs do not have the `<dds>` tag. Again we can just make the reader `wget` them

Reply:
Done